.github/dependabot.yml

```diff
@@ -7,18 +7,9 @@ updates:
     # Check the npm registry for updates every week on monday
     schedule:
       interval: "weekly"
-    reviewers:
-      - "julienw"
-      - "canova"
-      - "gregtatum"
-
-  # Enable version updates for Docker
-  - package-ecosystem: "docker"
-    # Look for a `Dockerfile` in the `root` directory
-    directory: "/"
-    # Check for updates once a week
-    schedule:
-      interval: "weekly"
+    # We have some linked dependencies (eg: babel) so let's bump up the limit of
+    # open PRs.
+    open-pull-requests-limit: 10
```

Comment:
I wish some day dependabot will send only one PR for all the dependency updates combined :)